setup_test_db.py

```diff
@@ -32,7 +32,8 @@
 `id` bigint NOT NULL AUTO_INCREMENT,
 `mongodb_id` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'Auto-generated id from MongoDB',
 `root_sample_id` varchar(255) COLLATE utf8_unicode_ci NOT NULL COMMENT 'Id for this sample provided by the Lighthouse lab',
-`cog_uk_id` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL COMMENT 'Consortium-wide id, generated by Sanger on import to LIMS',
+`cog_uk_id` varchar(255) COLLATE utf8_unicode_ci NOT NULL COMMENT 'Consortium-wide id, generated by Sanger on import to LIMS',
+`cog_uk_id_unique` tinyint(1) DEFAULT '1' COMMENT 'A flag indicating whether the COG UK ID value should be unique. NULL means no.',
```

Comment:
Has this new field been added already to the prod db?

Reply:
It will be when I finish populating the COG UK IDs that are taking a long time.  That will be start of next week.